q2_diversity/_beta/_method.py

```diff
@@ -11,6 +11,7 @@
 import skbio
 import skbio.diversity
 import skbio.tree
+import sklearn.metrics
```

Comment:
Sorry for the confusion -- we have a private (for now) draft doc that's on the forum detailing some of these qiime2 package requirements. I'll share that with you on Slack.